internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/QualysScannerConstants.java

```diff
@@ -0,0 +1,135 @@
+/*
+ *
```

Comment:
Remove this unnecessary line in the license header. Do the same thing for other places as well.
```suggestion
```